pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -152,6 +147,9 @@ edk2.mkDerivation projectDscPath (finalAttrs: {
   # release notes accordingly.
   postInstall = ''
     mkdir -vp $fd/FV
+  '' + lib.optionalString (builtins.elem fwPrefix [
+    "OVMF" "AAVMF" "RISCV_VIRT"
+  ]) ''
```

Comment:
This doesn't appear right to me. The fwPrefix is just being stuck on its own line in this script?

Reply:
The optional `mv` here does not work when building with `fwPrefix = "CLOUDHV"`. Please advise a better way how to make it optional.

Reply:
Yeah, it's fine. I see how I misread this last night.